### PR TITLE
Update admin config

### PR DESCRIPTION
**Pull request by DutchmanNL**
@HGlab01 Please review & test

### CODE REVIEW

File: admin/jsonConfig.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "visible": false,
+      "sm": 3,
+      "type": "password",
+      "label": "Password"
+    },
+    "intervall": {
+      "sm": 3,
+      "type": "number",
+      "label": "Interval in seconds",
+      "min" : 10,
```
Comment (HGlab01) [inline admin/jsonConfig.json:29]: in the adapter code 30 seconds is a minimum. We should take the 30 here as well

File: admin/jsonConfig.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "max" : 300,
```
Comment (HGlab01) [inline admin/jsonConfig.json:30]: is a max helpful? 300 is only 5 minutes. Maybe we should skip this one ore put it to e.g. 3600

**Review by HGlab01 — CHANGES_REQUESTED** [review]

**Comment by HGlab01** [comment]
https://github.com/DrozmotiX/ioBroker.tado/blob/updateAdminConfig/admin/index_m.html stays at it is?

File: admin/jsonConfig.json (hunk 2 above)
Comment (DutchmanNL) [reply admin/jsonConfig.json:30]: good point, not sure about the use cases but indeed updated it to 3600

File: admin/jsonConfig.json (hunk 1 above)
Comment (DutchmanNL) [reply admin/jsonConfig.json:29]: clear, updated

**Comment by DutchmanNL** [comment]
> https://github.com/DrozmotiX/ioBroker.tado/blob/updateAdminConfig/admin/index_m.html stays at it is?

good point, basically the this one is to support admin <5, if we remove it it means alll < Admin 5 install are not. able to update the configuration anymore.

so 2 options : 

- keep the file for legacy support and ensure any update is propagated to both
- Remove the content and make a kind of remark that this adapter config is only supported by admin 5 and also use that as dependency in io-package

**Comment by HGlab01** [comment]
If I put an invalide value to seconds the save is disabled --> ok
image
but if i put afterwards a valid value I have still no chance to save, also not if I change the email address afterwards
image

Maybe this is a framework issue?

**Comment by HGlab01** [comment]
> > https://github.com/DrozmotiX/ioBroker.tado/blob/updateAdminConfig/admin/index_m.html stays at it is?
> 
> good point, basically the this one is to support admin <5, if we remove it it means alll < Admin 5 install are not. able to update the configuration anymore.
> 
> so 2 options :
> 
> * keep the file for legacy support and ensure any update is propagated to both
> * Remove the content and make a kind of remark that this adapter config is only supported by admin 5 and also use that as dependency in io-package

ok, then let's keep both for a while!

**Comment by DutchmanNL** [comment]
> If I put an invalide value to seconds the save is disabled --> ok
> 
> image
> 
> but if i put afterwards a valid value I have still no chance to save, also not if I change the email address afterwards
> 
> image
> 
> 
> 
> Maybe this is a framework issue?

Interesting observation haven't seen that before.
Will test locally on Sunday

**Pull request merged**